src/Containers/Microsoft.NET.Build.Containers/LocalDaemons/DockerCli.cs

```diff
@@ -21,6 +21,7 @@ internal sealed class DockerCli : ILocalRegistry
 
     private readonly ILogger _logger;
     private string? _commandPath;
```

Comment:
nit: `_command` would be a better name than `_commandPath` with the refactorings I pushed here.

Reply:
good call, will rename